### PR TITLE
Update events for PP

### PR DESCRIPTION
**Pull request by ignasirv**
Add programVKey at `AddNewRollupType` and created new event `VerifyPessimisticTrustedAggregator` instead `VerifyBatchesTrustedAggregator` emitted when the trusted aggregator verifies a pessimistic proof

### CODE REVIEW

File: contracts/v2/PolygonRollupManager.sol [hunk 1]
```diff
@@ -292,6 +293,17 @@ contract PolygonRollupManager is
         address indexed aggregator
     );
 
+    /**
+     * @dev Emitted when the trusted aggregator verifies a pessimistic proof
+     */
+    event VerifyPessimisticTrustedAggregator(
```
Comment (invocamanman) [inline contracts/v2/PolygonRollupManager.sol:299]: same event helps the current bridge service to synchronize everything

Comment (ignasirv) [reply contracts/v2/PolygonRollupManager.sol:299]: Fixed!

**Review by krlosMata — APPROVED** [review]
:smile_cat:

File: docker/scripts/v2/deploy-docker.sh [hunk 2]
```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-sudo rm -rf docker/gethData/geth_data
```
Comment (invocamanman) [inline docker/scripts/v2/deploy-docker.sh:2]: sudo might be needed if docker is added at sudo level

Comment (krlosMata) [reply docker/scripts/v2/deploy-docker.sh]: added again `sudo`

**Review by invocamanman — APPROVED** [review]

**Review by invocamanman — APPROVED** [review]

File: docker/scripts/v2/deploy-docker.sh (hunk 2 above)
Comment (ignasirv) [reply docker/scripts/v2/deploy-docker.sh]: I don't think using sudo to run docker compose is a good approach

**Pull request merged**